src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -214,7 +214,13 @@ Vue.component('availbox', {
         + '<a :href="waiting_list_url" target="_blank" @click="$root.open_link_in_frame">' + strings.waiting_list + '</a>'
         + '</div>'
         + '<div class="pretix-widget-availability-available" v-if="!require_voucher && avail[0] === 100">'
-        + '<label class="pretix-widget-item-count-single-label" v-if="order_max === 1">'
+        + '<label class="pretix-widget-item-count-single-label pretix-widget-btn-checkbox" v-if="order_max === 1 && $root.single_item_select == \'button\'">'
+        + '<input type="checkbox" value="1" :checked="!!amount_selected" @change="amount_selected = $event.target.checked" :name="input_name"'
+        + '       v-bind:aria-label="label_select_item"'
+        + '>'
+        + '<i class="pretix-widget-icon-cart" aria-hidden="true"></i> Select'
```

Comment:
Are we purposefully using `<i>` and not span here? IIRC most people today advocate against using `<i>` for this these days

Reply:
Also, the "Select" is not translated here, is it?

Reply:
Oh sorry, totally missed the translation part. Just copied the generated HTML from pretix-presale. I think `<i>` is taken from fontawesome docs, but they state themselves, that `<i>` is used for bevity, but `<span>` would be more semantically correct. I will change that.